net/interfaces/interfaces.go

```diff
@@ -500,7 +500,7 @@ func isPrivateIP(ip netaddr.IP) bool {
 }
 
 func isGlobalV6(ip netaddr.IP) bool {
-	return v6Global1.Contains(ip)
+	return v6Global1.Contains(ip) || V6ULA.Contains(ip)
```

Comment:
maybe if we exclude our own at least?  but we might want to move this to tsaddr and exported, so we don't have different definitions in different places.

Reply:
Moved to tsaddr, and excluded our own.